mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/ProfileAssociationDao.java

```diff
@@ -53,15 +55,17 @@
    * Returns 'detail' profiles linked to 'master' profile id
    *
    * @param masterId 'master' profile id
+   * @param tenantId tenant id
    * @return future
    */
-  Future<D> getDetailProfilesByMasterId(String masterId);
+  Future<List<ChildSnapshotWrapper>> getDetailProfilesByMasterId(String tenantId, String masterId);
 
   /**
-   * Returns 'master' profiles linked to 'detail' profile id
+   * Returns master profiles linked to detail profile id
    *
-   * @param detailId 'detail' profile id
+   * @param tenantId tenant id
+   * @param detailId detail profile id
    * @return future
    */
-  Future<M> getMasterProfilesByDetailId(String detailId);
+  Future<List<ChildSnapshotWrapper>> getMasterProfilesByDetailId(String tenantId, String detailId);
```

Comment:
Please move 'tenantId' argument at the end of method arguments list

Reply:
Done